.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,6 @@
-<!--
-Any PR related to Mautic 2 issues is not relavant anymore, please consider upgrading your code to the Mautic 3 series (staging/3.0 branch).
--->
 | Q                                      | A
 | -------------------------------------- | ---
-| Branch?                                | staging for features or enhancements / 3.2 for bug fixes <!-- see below -->
+| Branch?                                | "features" for features or enhancements / 3.2 for bug fixes <!-- see below -->
```

Comment:
```suggestion
| Branch?                                | "features" for features or enhancements / 3.3 for bug fixes <!-- see below -->
```
We can as well bump the bug fix version.

Reply:
Wait... There won't be a branch for `3.2.1`, only a tag. The branch remains `3.2` for all 3.2.x related developments, right?.

Reply:
Oh, that make sense. I'm over-complicating things :) I'll delete the 3.2.1 and 3.3 branches for now.